src/components/breadcrumb/BreadcrumbLink/BreadcrumbLink.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+interface BreadcrumbLinkProps {
+  children: React.ReactNode
+  href: string
+  anchorProps?: JSX.IntrinsicElements['a']
+  spanProps?: JSX.IntrinsicElements['span']
```

Comment:
Maybe redundant prop? Not used in component

Reply:
Yep this was a lingering remnant of a previous iteration. It's being removed 😄

Reply:
Gone in 3b391bade03925a0ada15f4a2dd3881fb808dd3d